src/glyph.ts

```diff
@@ -88,8 +88,7 @@ class GlyphCache {
     const key = category ? `${code}%${category}` : code;
     let entry = entries[key];
     if (entry === undefined) {
-      const musicFontStack = Tables.MUSIC_FONT_STACK.slice();
-      entry = new GlyphCacheEntry(musicFontStack, code, category);
+      entry = new GlyphCacheEntry(Glyph.MUSIC_FONT_STACK, code, category);
```

Comment:
Yes, this is related to the above edit with `Glyph.MUSIC_FONT_STACK`.

It is related to breaking a circular dependency. Previously, the GlyphCache and Glyph class referenced the Tables.MUSIC_FONT_STACK. However, Tables also depends on the Glyph class because it calls `Glyph.getWidth()`. This is a direct circular dependency. It doesn't break the code currently, because at runtime both classes are already initialized by the time they need each other.

This change means that the Glyph and GlyphCache classes no longer know about Tables. Instead, every time the MUSIC_FONT_STACK is changed via the public API `Flow.setMusicFont(...)`, it will update both the Tables and Glyph / GlyphCache at the same time.

Reply:
Ok